src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
`enum` values should be uppercase. So please rename them to `BLACK`,  `WHITE`,  `RED`,  `BLUE`. If you really want to keep the lowercase, you can add a code like this:

    BLACK("black"), WHITE("white"), RED("red"), BLUE("blue");
    private final String val;

    private Color(String val) {
        this.val = val;
    }

    @Override
    public String toString() {
        return val;
    }